Configuration/Eras/python/Era_Run2_2017_lowPU_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_ppRef_cff import Run2_2017_ppRef
+from Configuration.Eras.Modifier_lowPU_2017_cff import lowPU_2017
+
+Run2_2017_lowPU = cms.ModifierChain(Run2_2017_ppRef, lowPU_2017)
```

Comment:
It may be more intuitive to call this Run2_2017_ppRef_noSRPF
or even Run2_2017_ppRef_lowPU to be more clear of the provenance of changes, which also include Run2_2017_ppRef